src/SmartComponents/Remediations/RemediationWizard.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import * as api from '../../api/remediations';
+import { Wizard } from '../../PresentationalComponents/Wizard';
+import Deferred from '../../Utilities/Deferred';
+
+import NameStep from './steps/Name';
+
+import './RemediationWizard.scss';
+
+let mountedInstance = false;
+
+export function getMountedInstance () {
+    return mountedInstance;
+}
+
+function createRemediation (name, { issues, systems }) {
+    name = (name && name.length) ? name : 'Unnamed remediation';
```

Comment:
This can be shortcutted to `name = name || 'Unnamed remediation';`

So the call can look like
```JS
return api.createRemediation({
        name: name || 'Unnamed remediation',
        add: {
            issues,
            systems
        }
    });
```